crates/rattler_conda_types/src/version/mod.rs

```diff
@@ -686,25 +687,29 @@ enum Component {
 
     /// An underscore or dash.
     UnderscoreOrDash {
+        /// Dash flag.
         is_dash: bool,
     },
 }
 
 impl Component {
+    /// Return a component as numeric value.
```

Comment:
```suggestion
    /// Returns a component as numeric value.
```

In Rust it is common practice to write doc comments in descriptive style:

```
Returns a component as numeric value.
```

instead of imperative:

```
Return a component as numeric value.
```